java/adapter/avro/src/main/java/org/apache/arrow/AvroToArrowUtils.java

```diff
@@ -78,69 +81,123 @@
    *   <li>BYTES --> ArrowType.Binary</li>
    * </ul>
    */
-  private static Field getArrowField(Schema schema, String name, boolean nullable) {
+  private static Consumer createConsumer(Schema schema, String name, BufferAllocator allocator) {
+    return createConsumer(schema, name, false, -1, allocator);
```

Comment:
make this a constant or use /*parameter_name=*/

Reply:
Thanks, fixed.